### ISSUE TITLE
sdig: Print exception instead of just crashing

### ISSUE THREAD

**Issue by zeha** [PowerDNS/pdns#3643]
```
# sdig localhost 53 deduktiva.com any
terminate called after throwing an instance of 'PDNSException'
Aborted
```

Please print that `localhost` is not an ip address instead.

**Comment by Habbie** [comment]
fixed in 1a4843e0a68693b5c7a6bdb26a41e3d6ca3a1a54 ?

**Comment by pieterlexis** [comment]
indeed fixed in master:
```
$ ./pdns/sdig localhost 53 deduktiva.com any
Fatal: Unable to convert presentation address 'localhost'
```
